src/WebUI/src/services/users-service.ts

```diff
@@ -8,12 +8,12 @@ import {
   type UserItemRank,
 } from '@/models/user';
 import Platform from '@/models/platform';
-import { type ClanEdition } from '@/models/clan';
+import { Clan, type ClanEdition, type ClanMemberRole } from '@/models/clan';
 import { type RestrictionWithActive } from '@/models/restriction';
 
 import { get, post, put, del } from '@/services/crpg-client';
 import { getActiveJoinRestriction, mapRestrictions } from '@/services/restriction-service';
-import { mapClanResponse } from '@/services/clan-service';
+import { mapClanResponse } from './clan-service';
```

Comment:
return it to the way it was before ;)
Or we will have to write that in all services